src/components/List.component.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// TODO: discuss if there are no outer dependecies should we wrap component into a function
```

Comment:
Should we wrap all the components in a function even if they don't have outer dependencies for consistency with those that will have?